scripts/cci-test-timings.data.json

```diff
@@ -0,0 +1,1070 @@
+{
+  "lastUpdated": "2022-12-31T02:03:46.162Z",
+  "totalTestFiles": 266,
+  "timingData": [
```

Comment:
timing data is the worst of either Linux or Windows median runtime for each file